src/main/scala/org/deepdive/extraction/ExtractorRunner.scala

```diff
@@ -400,7 +398,7 @@ class ExtractorRunner(dataStore: JsonExtractionDataStore, dbSettings: DbSettings
 
     // NEW: for mysqlimport compatibility, the file basename must be same as table name.
     val queryOutputFile = new File(queryOutputPath + s"${outputRel}.copy_query_${funcName}.tsv")
-    val gpFileName = s"${outputRel}_unload_${funcName}"
+    val gpFileName = s"${outputRel}_query_unload"
```

Comment:
So the loss of 'funcName' isn't crucial?  I am not sure where funcName comes from, so I can't tell if its loss will possibly cause a name clash here.